vignettes/demo.Rmd

```diff
@@ -95,33 +82,22 @@ The infrastructure files and directories that comprise a minimal R package are c
 fs::dir_tree()
 ```
 
-
 ### Create and configure a Git repository
 
-Next we set up this package for development and create a Git repository for the package.
-We use `git init` to get a predictable branch name.
-
-```{bash git}
-git init --initial-branch=main
-```
-
-In interactive mode, the `usethis::use_git()` function creates an initial commit, and the repository is in a clean state.
-In the demo this needs to be carried out manually.
+Next, one would set up this package for development and create a Git repository for the package.
+We achieved this with gert code.
 
-```{bash git-initial-commit}
-git add .
```

Comment:
For my own reference this is the thread seeing `git add .` reminded me of https://twitter.com/karissawhiting/status/1319680032617517057